malwarefront/src/components/AttributeUpdate.js

```diff
@@ -290,6 +291,7 @@ class AttributeUpdate extends Component {
             await api.addMetakeyDefinition(metakey, this.state.label, this.state.description, 
                                            this.state.template, this.state.hidden)
             this.handleUpdate()
+            this.setState({succes:true})
```

Comment:
Typo + error state must be cleaned up
```suggestion
            this.setState({success: true, error: null})
```